src/core/Akka/Actor/ActorCell.DefaultMessages.cs

```diff
@@ -429,6 +429,22 @@ public void Restart(Exception cause)
             SendSystemMessage(new Recreate(cause));
         }
 
+        /// <summary>
+        /// Overrideable in order to support issues such as https://github.com/petabridge/phobos-issues/issues/82
+        /// </summary>
+        protected virtual ActorStarted CreateActorStartedEvent()
```

Comment:
Going to customize these events mostly to support F# users, since Akkling actors all use the same base type, this should make them easier to distinguish in Phobos traces and metrics in the future.